app/graphql/types/invoices/object.rb

```diff
@@ -48,6 +49,10 @@ class Object < Types::BaseObject
 
       field :refundable_amount_cents, Integer, null: false
       field :creditable_amount_cents, Integer, null: false
+
+      def has_credit_notes
```

Comment:
Is it not possible to check `credit_notes` field instead?

Reply:
what do you mean ?